projects/ngx-prx-styleguide/src/lib/spinner/spinner.component.stories.ts

```diff
@@ -0,0 +1,164 @@
+import { storiesOf, moduleMetadata } from '@storybook/angular';
+import { action } from '@storybook/addon-actions';
+import { centered } from '@storybook/addon-centered/angular';
+import { withKnobs, text, array, boolean, number } from '@storybook/addon-knobs';
+import { SpinnerModule } from './spinner.module';
+
+// Module metadata for stories.
+const storiesModuleMetaData = moduleMetadata({
+  imports: [
+    SpinnerModule
+  ],
+  schemas: [],
+  declarations: [],
+  providers: [],
+});
+
+const storyStyles = `
+  .wrapper {
+    position: relative;
+    width: 100px;
+    height: 100px;
+    background: #0089bd;
+  }
+  .inverse {
+    background: none;
+  }
+  .force-scrolling {
+    margin: 50vh auto;
+  }
+`;
+
+storiesOf('Utilities|Spinner', module)
+  .addDecorator(withKnobs)
+  // .addDecorator(centered)
+  .addDecorator(storiesModuleMetaData)
+  .add(
+    'Usage Details (Knobs)',
+    () => {
+      const spinning = boolean('Spinning', true);
+      const delay = number('Delay', 300);
+      const inverse = boolean('Inverse', false);
+      const overlay = boolean('Overlay', false);
+      const overlayFixed = boolean('Overlay Fixed', false);
+      const loadingMessage = text('Loading Message', '');
+
+      return {
+        template: `
+          <div class="wrapper force-scrolling">
+            <prx-spinner
+              [spinning]="spinning"
+              [delay]="delay"
+              [inverse]="inverse"
+              [overlay]="overlay"
+              [overlayFixed]="overlayFixed"
+              [loadingMessage]="loadingMessage"
+            ></prx-spinner>
+          </div>
+        `,
+        props: {
+          spinning,
+          delay,
+          inverse,
+          overlay,
+          overlayFixed,
+          loadingMessage
+        },
+        styles: [storyStyles]
+      }
+    },
+    {
+      notes: {
+        markdown:`
+# Spinner
+
+The Spinner Component shows a loading indicator. Typically the spinner is
+displayed with \`*ngIf\` until a resource has finished loading.
+
+The animation uses \`position: absolute\`, so it should be inside a
+\`position: relative\` container.
```

Comment:
This `position: relative` needs to be in freakin bold or better yet delegated to the spinner component. I forget about it more often than I remember even though it's just basic positioning.